crates/starknet-devnet-server/src/restrictive_methods.rs

```diff
@@ -0,0 +1,119 @@
+use std::collections::HashMap;
+
+use lazy_static::lazy_static;
+lazy_static! {
+    static ref MAPPING_HTTP_URIS_TO_RPC_METHODS: HashMap<&'static str, &'static str> =
+        HashMap::from([
+            ("/dump", "devnet_dump"),
+            ("/load", "devnet_load"),
+            ("/set_time", "devnet_setTime"),
+            ("/increase_time", "devnet_increaseTime"),
```

Comment:
This is getting out of hand, each method name is hardcoded in four places in the code.

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/17b09cbe5d96dc6a88f42f058c5c34e17f4f7f35

Reply:
IMO the problem is not so much in hardcoding vs using constants. We need a solution that will allow us to specify the method name in a single place, but that's out of scope for this PR. Resolving.